seed/views/v3/column_mapping_profiles.py

```diff
@@ -0,0 +1,313 @@
+    "to_table_name: [one of the following: 'TaxLotState' 'PropertyState']"
+)
+
+
+class ColumnMappingProfileViewSet(OrgMixin, ViewSet):
+
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(
+            required=False,
+            description="Optional org id which overrides the users (default) current org id"
+        )],
+        request_body=AutoSchemaHelper.schema_factory(
+            {'preset_type': ['string']},
+            description="Possible Types: 'Normal', 'BuildingSync Default', BuildingSync Custom'"
+        )
+    )
+    @api_endpoint_class
+    @action(detail=False, methods=['POST'])  # POST in order to provide array/list
+    def filter(self, request):
+        """
+        Retrieves all profiles for an organization.
+        """
+        try:
+            preset_types = request.data.get('preset_type', [])
+            preset_types = [ColumnMappingPreset.get_preset_type(pt) for pt in preset_types]
+            filter_params = {'organizations__pk': self.get_organization(request, True).id}
+            if preset_types:
+                filter_params['preset_type__in'] = preset_types
+            profiles = ColumnMappingPreset.objects.filter(**filter_params)
+            data = [ColumnMappingPresetSerializer(p).data for p in profiles]
+
+            return JsonResponse({
+                'status': 'success',
+                'data': data,
+            })
+        except Exception as e:
+            return JsonResponse({
+                'status': 'error',
+                'data': str(e),
+            }, status=HTTP_400_BAD_REQUEST)
+
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(
+            required=False,
+            description="Optional org id which overrides the users (default) current org id"
+        )],
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'name': 'string',
+                'mappings': [{
+                    'to_field': 'string',
+                    'from_field': 'string',
+                    'from_units': 'string',
+                    'to_table_name': 'string',
+                }]
+            },
+            description="Optional 'name' or 'mappings'.\n" + mappings_description
+        )
+    )
+    @api_endpoint_class
+    def update(self, request, pk=None):
+        """
+        Updates a profile given appropriate request data. The body should contain
+        only valid fields for ColumnMappingPreset objects.
+        parameters:
+            - name: pk
+              description: ID of profile
+              required: true
+              paramType: path
+            - name: name
+              description: Name of profile
+              required: false
+              paramType: body
+            - name: mappings
+              description: List of dictionaries
+              required: false
+              paramType: body
```

Comment:
Do you think we can delete these old swagger docs here as well as in the other views?